dev-python/datrie/metadata.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
```

Comment:
not sure we can do this...
They need to agree with moving/copying to ::science
Or we can drop them as maintainer and only keep science.

Reply:
Oops, I thought I fixed that. Seems I messed up somewhere when copying things.